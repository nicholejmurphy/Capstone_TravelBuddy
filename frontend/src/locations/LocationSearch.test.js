import React from "react";
import ReactDOM from "react-dom";
import { render } from "@testing-library/react";
import LocationSearchForm from "./LocationSearch";
import { MemoryRouter } from "react-router";

// smoke test
it("renders without crashing", function () {
  const div = document.createElement("div");
  ReactDOM.render(
    <MemoryRouter>
      <LocationSearchForm setSearchTerm={null} searchTerm={""} />
    </MemoryRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

// snapshot test
it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <LocationSearchForm setSearchTerm={null} searchTerm={""} />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
