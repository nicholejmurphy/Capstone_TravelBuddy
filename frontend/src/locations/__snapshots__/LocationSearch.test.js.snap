// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="SearchForm mb-4 mt-4 ml-3 vw-75"
  >
    <form
      class=""
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <input
            class="shadow form-control"
            id="search"
            name="search"
            placeholder="Search"
            type="text"
            value=""
          />
        </div>
        <div
          class="col"
        >
           
          <button
            class="bg-info shadow btn btn-secondary"
          >
            Search
          </button>
        </div>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
